### PR TITLE
Added hardware construction to readme

### PR DESCRIPTION
**Pull request by breycarr**
* Added diagram of connection from pi to soil sensor
* linked to blog on connecting multiple slave devices

### CODE REVIEW

**Review by olliebrownlow — APPROVED** [review]
Fantastic - thanks Ben! Let's merge it

File: README.md [hunk 1]
```diff
@@ -27,6 +27,14 @@ D E V E L O P M E N T
 * Test framework: Pytest for Python3
 * Code Quality: Pylama and ESLint
 
+## Hardware Construction
+
+The 4 Pin JST Cable connects the Soil Moisture Sensor to pins 1 (power), 3 and 5 (I2C), and 9 (ground):
+
+![](https://github.com/breycarr/deep_root/blob/master/public/images/models/Sensor_hooked_up_to_Raspberry_Pi.jpg?raw=true)
+
+The LCD touchscreen covers all of these pins, so manual adjustment of the wires is necessary. A full explanation of how to connect the multiple slave devices can be found in [this blog post](https://medium.com/@makers_c_a_c_t_u_s/multiple-slave-devices-not-enough-rpi-gpio-pins-no-problem-c3403a981623)
```
Comment (t-ekla) [inline README.md]: I hate myself so much for being this pedantic but would it be possible to add a full stop at the end of this sentence (sorry, sorry!) ?

**Review by t-ekla — APPROVED** [review]
Great addition, thanks so much for adding this!

**Review by laurencetaylor — APPROVED** [review]
Like it, nice addition!

File: README.md (hunk 1 above)
Comment (breycarr) [reply README.md]: No problem at all, added and pushed

**Pull request merged**